### DISCUSSION TITLE
OR operator in projects view filter is broken

### DISCUSSION THREAD

**Discussion by jankrassnigg** [community/community#7749] [Projects and Issues]
In the Projects beta you can "filter" a view to show only certain items. For example using "status:Todo". The documentation says that you can use a whitespace as an AND operator and a comma as an OR operator.

The OR operator is broken, though. I have a view that uses this filter:
status:"In Progress",status:Planned

Instead of showing me all issues that are either "In Progress" or "Planned" it always only shows me the issues that match the FIRST condition. So in the filter above everything that would be "In Progress", but I can turn the condition around and then it would only show the "Planned" issues. Each condition alone does the right thing, but the OR operator doesn't work, at all.

Also this filter is not very forgiving when you insert a whitespace between the commas. It just doesn't work (or applies the AND rule) rather than telling you that "whitespace comma whitespace" is syntactically incorrect.

**Comment by tattan** [comment]
I believe they mean it works like the solution given to this issue: https://github.com/github/feedback/discussions/7643
